cardano-cli/test/cardano-cli-golden/files/golden/help/conway_governance_action_create-new-committee.cli

```diff
@@ -0,0 +1,79 @@
+Usage: cardano-cli conway governance action create-new-committee --governance-action-deposit NATURAL
```

Comment:
Why are there so many of these: 

```
                                                                   [ --stake-pool-verification-key STRING
                                                                   | --cold-verification-key-file FILE
                                                                   | --stake-pool-id STAKE_POOL_ID
                                                                   | --stake-verification-key STRING
                                                                   | --stake-verification-key-file FILE
                                                                   | --stake-key-hash HASH
                                                                   ]
```

I think it needs to take something like 
```
--new-committee-member–cold-key-hash STRING
```

Reply:
Users can submit their stake key in different formats, via the hash, the key string or the file.

Reply:
@Jimbo4350 

I know,  but it shows the same thing 3 times

                                                                   ( --stake-pool-verification-key STRING
                                                                   | --cold-verification-key-file FILE
                                                                   | --stake-pool-id STAKE_POOL_ID
                                                                   | --stake-verification-key STRING
                                                                   | --stake-verification-key-file FILE
                                                                   | --stake-key-hash HASH
                                                                   )
                                                                   [ --stake-pool-verification-key STRING
                                                                   | --cold-verification-key-file FILE
                                                                   | --stake-pool-id STAKE_POOL_ID
                                                                   | --stake-verification-key STRING
                                                                   | --stake-verification-key-file FILE
                                                                   | --stake-key-hash HASH
                                                                   ]
                                                                   [
                                                                     ( --stake-pool-verification-key STRING
                                                                     | --cold-verification-key-file FILE
                                                                     | --stake-pool-id STAKE_POOL_ID
                                                                     | --stake-verification-key STRING
                                                                     | --stake-verification-key-file FILE
                                                                     | --stake-key-hash HASH
                                                                     )
                                                                     --epoch NATURAL]
                                                                   --quorum RATIONAL
                                                                   --tx-in TX-IN
                                                                   --out-file FILE

Reply:
They are not the same thing:

deposit return stake address, old committee stake keys, new committee stake keys with expiration epoch

Reply:
They do need to be updated to reflect this however as it's not obvious.